jdatetime/__init__.py

```diff
@@ -9,6 +9,7 @@
 import datetime as py_datetime
 import locale as _locale
 import re as _re
+from functools import partial as _partial
```

Comment:
Why did you import `partial` as  `_partial`?

Reply:
PEP8's convention for "internal use" (private) variables. It indicates that the variable is not part of the stable API and if they `from jdatetime import _partial`, although it works, it may break without deprecation in future versions. I guess that's also why they have imported `re as _re` in the previous line.

Reply:
Yes, but I am not sure this should be applied to standard libraries as well.
at least I haven't seen it before.
BTW, let's keep it like that.

Reply:
that convention is about defining new terms (funcs, vars, etc.).  Are we also using `_sys` and `_datetime`?
the reason `locale` is being imported like that here, is that there were other local variables named `locale`, so to avoid name clashes it was renamed as `_locale`.